CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# v7.0.0a1
+**NOTE:**
+- We are doing an Alpha release for v7.0.0 because there will be an **ACTION REQUIRED** API breaking change in this client to support Python 3.7, which substitutes the currently being-used `async` reserved keyword [kubernetes-client/gen#67](https://github.com/kubernetes-client/gen/pull/67)
+
```

Comment:
so the PR has not been merged, do you want to wait for it?

Reply:
I'd like to release v7 alpha first, and we can release v7 stable afterwards to introduce the breaking change. What do you think?